README.md

```diff
@@ -35,6 +36,7 @@ integration with *gnome-shell* or *Unity*, install `solaar-gnome3`.
 * a [Mageia package][mageia], courtesy of David Geiger
 * an [OpenSUSE rpm][opensuse], courtesy of Mathias Homann
 
+[ppa1]: https://launchpad.net/~solaar-unifying/+archive/ubuntu/ppa
```

Comment:
I think that a link to the Stable PPA is better here https://launchpad.net/~solaar-unifying/+archive/ubuntu/stable